packages/asc-ui/src/components/FormTitle/FormTitleStyle.ts

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components'
+import { themeColor, themeSpacing } from '../../utils/themeUtils'
+
+export default styled.h1`
+  max-width: 620px;
+  margin: ${themeSpacing(6, 0, 8)};
+  padding-bottom: 6px;
```

Comment:
We have to assume the designers were using a 5 point grid here, so this should be 1 `themeSpacing` unit on our code.

Reply:
fixed

Reply:
:ok: